ci/task_e2e_tests.yaml

```diff
@@ -17,20 +17,17 @@ run:
     - -c
     - |
       set -o pipefail
-      SENTRY_CLI=$PWD/cache/sentry-cli-1.37.0
+      SENTRY_CLI=/usr/local/bin/sentry-cli
```

Comment:
If we do it like this then we can also remove the `cache` stanza from the task definition

Reply:
@databus23 I changed this.